Adds a `-w` option to throttle commands execution

HorlogeSkynet opened the pull request:
Hey over here !

This is a pretty simple patch implementing a new `-w` option to throttle commands execution (non-interactive mode).
It could be pretty useful to `say` a message to players before `stop`-ping the server (for instance).

Bye 👋

Tiiffi commented:
Hello and thank you.

This seems reasonable addition but I would like to merge this to develop branch for now. Can you make pull requests for develop branch?

Also `sleep` function is not portable across different platforms so I would suggest creating portable (should work at least on Linux, MacOS and Windows) wrapper for sleep. Something like this should work (not tested yet):

```C
unsigned int mcrcon_sleep(unsigned int time)
{
        unsigned int ret = time;

	#ifdef _WIN32
            Sleep(time * 1000);
        #endif
        ret = sleep(time);

        return ret;
}
```
Also I think `atoi` is a bit unsafe (it can overflow and that is undefined behaviour as far as I know).

I would suggest replacing it with `strtol` implementation or at least make checks for `atoi` return value (should be >0 and suitable value for sleep function).

HorlogeSkynet commented:
Hey back @Tiiffi, thanks for your fast feedback !

So I've taken in consideration your comments :

1. My `master` is now your `develop` with the patch on top of it, and this very PR has been changed to update your `develop` directly ;
2. At the moment there is only one `sleep` occurrence so I've added the macro for Windows' case directly in the `run_commands` function ;
3. A proper `strtol` call now replaces the first naive `atoi` one ;
4. [BONUS] I've updated as well the `run_commands` function, simplifying its logic mostly.

Bye, keep me posted 👋

Tiiffi commented:
Hello again @HorlogeSkynet  

I checked the changes and everything looks good to me. I will merge this pull request.

Thank you.
